templates/components/loaders/python/db.py

```diff
@@ -0,0 +1,29 @@
+import os
+import logging
+from typing import List
+from pydantic import BaseModel, validator
+from llama_index.core.indices.vector_store import VectorStoreIndex
+
+logger = logging.getLogger(__name__)
+
+
+class DBLoaderConfig(BaseModel):
+    uri: str
+    queries: List[str]
+
+
+def get_db_documents(raw_configs: list[dict]):
```

Comment:
why not separating parse the config and using the config?
```suggestion
def get_db_documents(raw_configs: list[DBLoaderConfig]):
```